stable/kommander/Chart.yaml

```diff
@@ -9,4 +9,4 @@ maintainers:
   - name: gracedo
   - name: dkoshkin
 name: kommander
-version: 0.30.0
+version: 0.30.1
```

Comment:
```suggestion
version: 0.31.0
```
minor version bump to match UI minor bump?

Reply:
sure.